### PR TITLE
rename `peak_signal_noise_ratio`

### PR DESCRIPTION
**Pull request by Fariborzzz**
## What does this PR do?

Change 
'PSNR' to 'PeakSignalNoiseRatio'
`psnr` to `peak_signal_noise_ratio`

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Yep :smile: 

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Fariborzzz** [comment]
Linked to issue: Unify functional metrics names #729

**Comment by SkafteNicki** [comment]
Hi @Fariborzzz, it seems this PR is changing a lot more file than intended. Could you fix that?

**Comment by Borda** [comment]
> Hi @Fariborzzz, it seems this PR is changing a lot more file than intended. Could you fix that?

the case seems to be that you have by default some different file permissions which yielded in all project changes, for next time is better to set your git to ignore file permissions before committing changes

**Comment by Borda** [comment]
@Fariborzzz shall be fixed now :rabbit:

**Comment by Fariborzzz** [comment]
> the case seems to be that you have by default some different file permissions which yielded in all project changes, for next time is better to set your git to ignore file permissions before committing changes

Thank you @SkafteNicki and @Borda. Sure, I'll set up git ignore for my next commits 😃!
This was the first commit, hope the next ones are smoother.

**Comment by SkafteNicki** [comment]
Hi @Fariborzzz,
Could you also rename the modular class:
`PSNR` -> `PeakSignalNoiseRatio`
?

**Review by Borda — CHANGES_REQUESTED** [review]
deprecation warnings & back compatibility

**Comment by Fariborzzz** [comment]
Fbeta_score is also added to this PR.

**Comment by Borda** [comment]
> Fbeta_score is also added to this PR.

lets rater open it in separate as there is not much logical connection between these two metrics and you get more contribution ;)

File: torchmetrics/functional/image/psnr.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def _psnr_update(
     return sum_squared_error, n_obs
 
 
-def psnr(
+def peak_signal_noise_ratio(
```
Comment (Borda) [inline torchmetrics/functional/image/psnr.py:94]: need to keep the original `psnr` for back compatibility

File: torchmetrics/image/psnr.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 from torchmetrics.utilities import rank_zero_warn
 
 
-class PSNR(Metric):
+class PeakSignalNoiseRatio(Metric):
```
Comment (Borda) [inline torchmetrics/image/psnr.py:24]: need to keep the original `PSNR` for back compatibility

**Comment by Borda** [comment]
@Fariborzzz  could you pls resolve the conflict or shall I do it for you? :rabbit:

File: torchmetrics/functional/image/psnr.py (hunk 1 above)
Comment (Fariborzzz) [reply torchmetrics/functional/image/psnr.py:94]: > @Fariborzzz could you pls resolve the conflict or shall I do it for you? rabbit

I appreciate your help, a bit confused with my first commit :rabbit2: . Hope backward compatibility works fine now.

**Review by Borda — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Pull request merged**